util.js

```diff
@@ -55,33 +42,15 @@ exports.tarHeaderChecksumMatches = buffer => { // Does not check if checksum fie
 	);
 };
 
-exports.multiByteIndexOf = (buffer, bytesToSearch, startAt = 0) => {
-	// `Buffer#indexOf()` can search for multiple bytes
-	if (Buffer && Buffer.isBuffer(buffer)) {
-		return buffer.indexOf(Buffer.from(bytesToSearch), startAt);
-	}
-
-	const nextBytesMatch = (buffer, bytes, startIndex) => {
-		for (let i = 1; i < bytes.length; i++) {
-			if (bytes[i] !== buffer[startIndex + i]) {
-				return false;
-			}
-		}
-
-		return true;
-	};
-
-	// `Uint8Array#indexOf()` can search for only a single byte
-	let index = buffer.indexOf(bytesToSearch[0], startAt);
-	while (index >= 0) {
-		if (nextBytesMatch(buffer, bytesToSearch, index)) {
-			return index;
-		}
-
-		index = buffer.indexOf(bytesToSearch[0], index + 1);
-	}
+exports.uint8ArrayUtf8ByteString = uint8ArrayUtf8ByteString;
 
-	return -1;
+/**
+ * ID3 UINT32 sync-safe tokenizer token
+ * 28 bits (representing up to 256MB) integer, the msb is 0 to avoid 'false syncsignals'
+ */
+exports.uint32SyncSafeToken = {
+	get: (buf, off) => {
+		return (buf[off + 3] & 0x7F) | ((buf[off + 2]) << 7) | ((buf[off + 1]) << 14) | ((buf[off]) << 21);
+	},
+	len: 4
```

Comment:
```suggestion
	length: 4
```

Reply:
Sorry, this part if the [token interface](https://github.com/Borewit/token-types/blob/de1957685596a1596df5daae4afaf89dbd709522/lib/index.ts#L28).
Cannot do that, will break the token.

Reply:
Alright. No worries. Although, I would recommend taking a look at https://github.com/sindresorhus/eslint-plugin-unicorn/blob/master/docs/rules/prevent-abbreviations.md It really makes code more readable to not use abbreviations.

Reply:
I agree, despite the fact I cheat sometimes for good or bad reasons. I am firm believer in readable code. If that takes a bit more text that is fine. There are ways to optimize, if that is the reason to deviate from write verbose code. Easily being able to understand what is going on is one of them.

I did not design the token interface btw, I inherited it via [strtok2](https://www.npmjs.com/package/strtok2) (which I own and maintained and extended, but I did not write it initially). I wrote [strtok3](https://www.npmjs.com/package/strtok3) replacing the events with promises which allowed the _tokens_ to be typed. The _tokens_ remained intact.